sunpy/coordinates/tests/test_sun.py

```diff
@@ -263,7 +265,9 @@ def test_carrington_rotation_number(date, day_fraction, rotation_number):
                           ])
 def test_carrington_rotation_starttime(crot, julian_days):
     # Stated precision in the docstring is 0.11 seconds
-    assert_quantity_allclose(sun.carrington_rotation_time(crot).tt.jd * u.day, julian_days * u.day, atol=0.11*u.s)
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore", category=ErfaWarning)
```

Comment:
`ErfaWarning` is a very broad category unfortunately. What exactly are you catching here?

Reply:
> `ErfaWarning` is a very broad category unfortunately.

As far as I know, `ErfaWarning` is virtually always the result of providing "dubious" times to ERFA, where "dubious" is defined by being too far in the past or too far in the future in ERFA's judgment.  Here, it's because the test dates are all far in the past.

Reply:
That certainly seems to be the most common one, but in theory it can be a lot more (any warning from the erfa layer at all iirc.).